package.json

```
@@ -5,6 +5,7 @@
   "main": "./src/service/service.js",
   "scripts": {
     "start": "nodemon ./src/service/service.js",
+    "start-front-server": "nodemon ./src/express/express.js",
```

лучше задавать порт как переменную среды и потом использовать его в коде
```suggestion
    "start-front-server": "PORT=8080 nodemon ./src/express/express.js",
```